WarehouseConfiguration warehouse_configuration
PackageConfiguration[] package_configurations
---
bool success

